src/common/commandExecutor.ts

```diff
@@ -37,13 +39,23 @@ export enum CommandStatus {
 export class CommandExecutor {
 
     private static ReactNativeCommand = "react-native";
-    private static ReactNativeVersionCommand = "--version";
     private childProcess = new Node.ChildProcess();
+    private static ReactNativeGlobalCommandName = "";
```

Comment:
Why static?

Reply:
This property is used for all CommandExecutor's entities. CommandExecutor's objects are created in many places, so we should assign one value(`ReactNativeGlobalCommandName`) for all objects. Correct me please if I'm wrong.

Reply:
Can we use a `ReactNativeCommand` variable for that? I think we can also make it `public static` and get rid of the setter.

Reply:
Yes, we can. I removed that static setter and changed the behavior of setting `ReactNativeCommand`.